public/js/Route/bbRoute.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+/* eslint-disable comma-dangle */
+/* eslint-disable no-console */
+
+(function(){
+  window.App = {
```

Comment:
You're defining App in two places, so one of the definitions is being overridden. window.App is the same thing as var App down below, because you're operating in the context of window as your global object. I'm guessing this is the one that's being overwritten, so you'll want to remove this iife here.